integrations/langflow/tests/helpers/testing/stepflow_binary.py

```diff
@@ -152,6 +153,162 @@ def is_healthy(self) -> bool:
         except Exception:
             return False
 
+    def store_flow(self, flow: dict[str, Any], timeout: float = 30.0) -> str | None:
```

Comment:
Double check these new test helpers are used / needed. Check whether any of the old helpers can be removed.